forge-ai/src/main/java/forge/ai/AiAbilityDecision.java

```diff
@@ -0,0 +1,9 @@
+package forge.ai;
+
+public record AiAbilityDecision(int rating, AiPlayDecision decision) {
```

Comment:
<img width="299" height="138" alt="image" src="https://github.com/user-attachments/assets/8db53cdf-427a-48ac-a2bb-ec74c59ac1b9" />

Reply:
No I tested this. The syntactic sugar is in the Build Tools we use:

https://github.com/Card-Forge/forge/blob/11815a4cf2e395c5d41fcea40465379b1285aaa2/forge-core/src/main/java/forge/card/CardEdition.java#L210